src/metabase/embedding/api/theme.clj

```diff
@@ -0,0 +1,75 @@
+(ns metabase.embedding.api.theme
+  "Endpoints for managing embedding themes."
+  (:require
+   [metabase.api.common :as api]
+   [metabase.api.macros :as api.macros]
+   [metabase.util.malli.registry :as mr]
+   [metabase.util.malli.schema :as ms]
+   [toucan2.core :as t2]))
+
+(set! *warn-on-reflection* true)
+
+(mr/def ::EmbeddingTheme
+  [:map
+   [:id       ms/PositiveInt]
+   [:name     ms/NonBlankString]
+   [:settings :map]
+   [:created_at  :any]
```

Comment:
`ms/TemporalString` is returning an error here, I think toucan is returning an actual java Temporal object and it's not really a string

Reply:
Thank you! I didn't expect to literally have to reference the Temporal class (I was expecting the schema to have something like `ms/Temporal`) but I guess that's Java for you. Let me swap this 🙏🏻

Reply:
Fixed in https://github.com/metabase/metabase/pull/65373/commits/ae8b471a6411d086ecb0037ff7f800012c6edf54 🙏🏻